husky_components/datagrid/main.js

```diff
@@ -2175,6 +2184,10 @@
                         url = this.sandbox.uritemplate.expand(uriTemplate, {page: page, limit: limit});
                     }
 
+                    if (this.viewId === 'table') {
```

Comment:
because for other decorator you have to load all the other pages also (e.g. masonry) so i and @chirimoya decided to do it only for table in the first version.